Automattic/jetpack
# Jetpack Cloud Simple > Monetize: Fix the link for "Set up an offer for your supporters"

## Proposed changes:


This PR fixes the link for "Set up an offer for your supporters" on Jetpack Cloud Simple > Monetize, since we [enabled](https://github.com/Automattic/wp-calypso/pull/89508) Jetpack Cloud for Simple. 

This PR modifies the condition that was added in https://github.com/Automattic/jetpack/pull/36728 and https://github.com/Automattic/jetpack/pull/36573 so that it shows the link to Jetpack Cloud on Simple. I assume it is safe since users still can't set the admin interface Classic or access Jetpack Cloud. 

Fixes https://github.com/Automattic/dotcom-forge/issues/7333
Related to https://github.com/Automattic/wp-calypso/pull/91362

<img width="1050" alt="Screenshot 2024-06-03 at 14 18 03" src="https://github.com/Automattic/jetpack/assets/5287479/81b97d89-76a8-4387-8d25-6ac7328f7fc6">

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply this patch to your sandbox https://github.com/Automattic/jetpack/pull/37673#issuecomment-2144290684
* Start your local `http://jetpack.cloud.localhost:3000/` with `yarn start-jetpack-cloud`
* Prepare a Simple Classic site
	* Go to `http://jetpack.cloud.localhost:3000/monetize/<your-simple-classic-site>`
	* Ensure the "Set up an offer for your supporters" link points to `/monetize/payments`
	* Go to `https://wordpress.com/earn/<your-simple-classic-site>`
	* Ensure the "Set up an offer for your supporters" link points to `/monetize/payments`
		* This is the same behavior as Atomic Classic. We may need a broader [discussion](https://github.com/Automattic/wp-calypso/pull/91362#pullrequestreview-2092854525).
* Prepare a Simple Default site 
	* Go to `https://wordpress.com/earn/<your-simple-classic-site>`
	* Ensure the "Set up an offer for your supporters" link points to `/earn/payments`
